test/libyul/Common.cpp

```diff
@@ -91,7 +91,7 @@ std::pair<std::shared_ptr<Object>, std::shared_ptr<yul::AsmAnalysisInfo>> yul::t
 yul::Block yul::test::disambiguate(std::string const& _source, bool _yul)
 {
 	auto result = parse(_source, _yul);
```

Comment:
What does ``_yul`` change here exactly?

Reply:
switching between `YulStack::Language::StrictAssembly` -> untyped and `YulStack::Language::Yul` -> typed